### PR TITLE
Calibration:add a new vehicle configuration files

### PR DESCRIPTION
**Pull request by cuishiwei**

### CODE REVIEW

File: modules/calibration/data/dev_kit_standard/vehicle_info.pb.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name : "CH"
+brand : CH
+model : CH
+license {
+    vin : "ch05"
```
Comment (macDure) [inline modules/calibration/data/dev_kit_standard/vehicle_info.pb.txt:5]: change the devkit01

File: modules/calibration/data/dev_kit_standard/gnss_conf/gnss_conf.pb.txt [hunk 2]
```diff
@@ -0,0 +1,83 @@
+data {
+    format: NOVATEL_BINARY
+    serial {
+        device: "/dev/ttyACM0"
+        baud_rate: 460800
```
Comment (macDure) [inline modules/calibration/data/dev_kit_standard/gnss_conf/gnss_conf.pb.txt:5]: change it to 115200

**Review by diegohu — CHANGES_REQUESTED** [review]
Add modules/calibration/data/dev_kit/cancard_params/canbus.conf and remove modules/calibration/data/dev_kit_standard/cancard_params/conti_radar_conf.pb.txt

File: modules/calibration/data/dev_kit_standard/camera_params/front_12mm_extrinsics.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+transform:
+  rotation:
+    x: -0.5090692569566782
+    y: 0.5096539036039291
+    z: -0.4782621974758038
+    w: 0.5023610859018319
+  translation:
+    x: 0.6222435832023621
+    y: -0.1885032951831818
+    z: -0.6547802686691284
```
Comment (SeasoulChris) [inline modules/calibration/data/dev_kit_standard/camera_params/front_12mm_extrinsics.yaml:17]: It is better to give default external parameters according to dev_kit_standard

File: modules/calibration/data/dev_kit_standard/camera_params/front_6mm_extrinsics.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+transform:
+  rotation:
+    x: 0.4980764736112689
+    y: -0.5019161381941862
+    z: 0.5019161381941863
+    w: -0.4980765070503043
+  translation:
+    x: 0.4592351019382477
+    y: -0.009229115210473537
+    z: -0.75
```
Comment (SeasoulChris) [inline modules/calibration/data/dev_kit_standard/camera_params/front_6mm_extrinsics.yaml:17]: It is better to give default external parameters according to dev_kit_standard

File: modules/calibration/data/dev_kit_standard/velodyne_params/velodyne16_novatel_extrinsics.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+transform:
+  rotation:
+    x: 0.007346950254289652
+    y: -0.00445628298530136
+    z: 0.7062893750216824
+    w: 0.7078710917924231
+  translation:
+    x: 0.01242604040985907
+    y: 0.2883885694403238
+    z: 0.8
```
Comment (SeasoulChris) [inline modules/calibration/data/dev_kit_standard/velodyne_params/velodyne16_novatel_extrinsics.yaml:17]: It is better to give default external parameters according to dev_kit_standard

**Review by diegohu — APPROVED** [review]

**Pull request closed without merging**